### ISSUE TITLE
Add support for 1.35?

### ISSUE THREAD

**Issue by jdlrobson** [Liquipedia/flow#15]
When trying to load the skin on 1.35 I get:

```
Fatal error: Uncaught ExtensionDependencyError: LiquiFlow is not compatible with the current MediaWiki core (version 1.36.0-alpha), it requires: >= 1.31.0, <= 1.32.9999. in /Users/jrobson/git/core/includes/registration/ExtensionRegistry.php:404 Stack trace: #0 /Users/jrobson/git/core/includes/registration/ExtensionRegistry.php(228): ExtensionRegistry->readFromQueue(Array) #1 /Users/jrobson/git/core/includes/Setup.php(161): ExtensionRegistry->loadFromQueue() #2 /Users/jrobson/git/core/includes/WebStart.php(89): require_once('/Users/jrobson/...') #3 /Users/jrobson/git/core/index.php(44): require('/Users/jrobson/...') #4 {main} thrown in /Users/jrobson/git/core/includes/registration/ExtensionRegistry.php on line 404
```

As  a result  https://www.mediawiki.org/wiki/Skin:LiquiFlow  has been marked as "unstable".

If this skin is maintained and you are willing to accept pull request please let me know (Note that patches would retain the existing support for 1.32) !

**Comment by FO-nTTaX** [comment]
I was not even aware that there was a mediawiki.org page for the skin.

Liquipedia has ceased usage of this skin (although it is still installed as we are currently on MW 1.31 LTS, the skin will be removed from our wiki when we have upgraded to MW 1.35 LTS). I was aware of it not working in MW >= 1.33 which is why I had marked it as such in `skin.json`. I am willing to accept pull requests to the skin, but there will not be any further support from our team for this skin beyond accepting pull requests. I would suggest considering this skin as unmaintained unless someone else wants to take over (in which case we would want removed all Liquipedia/Team Liquid branding currently in image files, localization files and the footer, a rename might be appropriate in that case too).
